astroplan/plots/__init__.py

```diff
@@ -1,2 +1,7 @@
+"""
+`astroplan.plots` is a module for making plots of important quantities within
+the `astroplan` toolbox, such as airmass vs. time, etc.
```

Comment:
I'd suggest you mention that, at least at the moment, we only support matplotlib, maybe this:
```
`astroplan.plots` contains functions to create common astro planning plots using `matplotlib`.
```